src/com/TTT/Board.java

```diff
@@ -7,20 +7,25 @@
 
     private String[] spaces;
 
-    public Board() {
-        spaces = new String[]{"1","2","3","4","5","6","7","8","9"};
+    public Board(int size) {
+        if(size == 4) {
+            spaces = new String[]{"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16"};
+        }
+        else {
+            spaces = new String[]{"1","2","3","4","5","6","7","8","9"};
+        }
```

Comment:
Still not happy with using an array of strings for this - it was the simplest thing at the time but I seem to have missed a more opportune point at which to refactor. I think I will probably refactor the use of X and O as strings first, to make this bit easier.